### PR TITLE
add functionality for working with mtxLora

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

**Review by DarkPark — CHANGES_REQUESTED** [review]

File: src/js/constants/accessKey.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import * as joobyCodec from 'jooby-codec';
+
+
+export const LENGTH_BYTES = 16;
+export const DEFAULT = joobyCodec.utils.getHexFromBytes(
```
Comment (DarkPark) [inline src/js/constants/accessKey.js:5]: DEFAULT_BYTES and DEFAULT_HEX ?

File: .vscode/settings.json [hunk 2]
```diff
@@ -6,7 +6,8 @@
         "fontsource",
         "roboto",
         "noconflict",
-        "rulesdir"
+        "rulesdir",
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply .vscode/settings.json]: Deleted

File: src/js/constants/severityTypes.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const SUCCESS = 'success';
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/js/constants/severityTypes.js:4]: Discussed, added a comment

File: src/js/constants/accessKey.js (hunk 1 above)
Comment (aleks-and) [reply src/js/constants/accessKey.js]: Renamed DEFAULT to DEFAULT_HEX, DEFAULT_BYTES will not be used

**Review by DarkPark — APPROVED** [review]

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**